src/main/java/com/feedzai/commons/sql/abstraction/engine/impl/OracleEngine.java

```diff
@@ -457,31 +457,7 @@ protected void createTable(final DbEntity entity) throws DatabaseEngineException
      * @since 2.1.13
      */
     private String getCreateTableStatement(final DbEntity entity) throws DatabaseEngineException {
-        final List<String> createTable = new ArrayList<>();
-
-        createTable.add("CREATE TABLE");
-        createTable.add(quotize(entity.getName()));
-        final List<String> columns = new ArrayList<>();
-        for (final DbColumn c : entity.getColumns()) {
-            final List<String> column = new ArrayList<>();
-            column.add(quotize(c.getName()));
-            column.add(translateType(c));
-
-            if (c.isDefaultValueSet() && !c.getDbColumnType().equals(DbColumnType.BOOLEAN)) {
-                column.add("DEFAULT");
-                column.add(translate(c.getDefaultValue()));
-            }
-
-            for (final DbColumnConstraint cc : c.getColumnConstraints()) {
-                column.add(cc.translate());
-            }
-            columns.add(join(column, " "));
-        }
-        createTable.add("(" + join(columns, ", ") + ")");
-        // segment creation deferred can cause unexpected behaviour in sequences
-        // see: http://dirknachbar.blogspot.pt/2011/01/deferred-segment-creation-under-oracle.html
-        createTable.add("SEGMENT CREATION IMMEDIATE");
-        return join(createTable, " ");
+        return translator.translateCreateTable(entity);
     }
```

Comment:
Also, on the `createTable`method, right now we have this:
```
defaultCreateTableStatement = get createTable from translator
createTableCompressLOBS = getCompressLobsStatements(entity);
createTableStatement = defaultCreateTableStatement + createTableCompressLOBS;
```
Then we try to create the table with `createTableStatement`, which may or may not contain LOB options depending on PDB config to compress LOBs (if the option is disabled, this statement is the same as `defaultCreateTableStatement`).
If this create table fails because the DB doesn't support LOBs, then we have a retry with the default statement (without LOBs).

I would propose a change to this logic: have a single statement to create tables, with or without LOB options according to the config. This statement would be produced at `translator.translateCreateTable(entity)` as it is now, but would include the LOB options (if applicable).
This would make the logic much simpler, the method `getCompressLobsStatements` would go away and its code integrated in the `translateCreateTable`.

As it is now, with "compress LOBs" enabled PDB will either create the table with LOB compression, or if the DB doesn't support it, will create the table without compression.... with the proposed change, in the same scenario PDB would will either create the table with LOB compression or fail.
However, I believe that this is acceptable: it is up to whoever sets up the environment to make sure that the DB supports LOB compression, or if it doesn't, configure PDB with LOB compression disabled.
What do you think?
Also, @jcsf @joaonuno